### PR TITLE
Add Line 6

### PR DESCRIPTION
**Pull request by hispanasian**

### CODE REVIEW

**Comment by phallobst** [comment]
The image is named `Line6.com.svg` (capital L) instead of `line6.com.svg`. The `"img"` property must be deleted as well here, since it's the default value anyway.

**Comment by hispanasian** [comment]
> The image is named `Line6.com.svg` (capital L) instead of `line6.com.svg`. The `"img"` property must be deleted as well here, since it's the default value anyway.

Thanks for the tips. I took a bit too long to fix the capital `L` here on windows. Wasn't aware I could delete the `img` property, but that explains why it's not on most files. Perhaps it can be removed from the `CONTRIBUTING.MD` example where `img` is explicitly declared unless there's a good reason to keep it there?

**Comment by Carlgo11** [comment]
@hispanasian the only example in [CONTRIBUTING.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md) that includes an `img` tag is the example in _"[New Sites](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#new-sites)"_ showcasing the available elements and that section even contains the following text:
>The default value for the icon is `<domain>.svg`, but can be overridden by an `img` field.

If you have any suggestions on how to phrase it clearer please let us know :wink:

File: img/l/line6.com.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
...
+</svg>
```
Comment (Carlgo11) [inline img/l/line6.com.svg]: Here's an inverted version of the one you provided. It seems it's the version used on their website.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1017.2 201.7"><path d="M32 185.8c13.2.8 229.9 14.7 476.6 16 246.7-1.3 463.4-15.2 476.5-16 26-2.9 32-14.4 32-32.1V48c0-17.7-6-29.2-32-32C972 15 755.3 1.3 508.6 0A8823.8 8823.8 0 0 0 32 16C6 18.8 0 30.3 0 48v105.7c0 17.7 6 29.2 32 32z"/><path fill="#fff" d="M32 185.8c13.2.8 229.9 14.7 476.6 16 246.7-1.3 463.4-15.2 476.5-16 26-2.9 32-14.4 32-32.1V48c0-17.7-6-29.2-32-32C972 15 755.3 1.3 508.6 0A8823.8 8823.8 0 0 0 32 16C6 18.8 0 30.3 0 48v105.7c0 17.7 6 29.2 32 32zm900.7-70.4c-.4-7.8-16.7-12-36.8-11.4-20 .5-35.9 5.5-35.5 13.3.3 7.8 16.6 11.6 36.7 11 20-.5 36-5 35.6-13zm29-55.1c2.6-.1-13.1 21.8-13.1 21.8s-17.3-9.7-45.2-9c-30 .8-53.2 8.8-52.8 25A129 129 0 0 1 905 86.7c34.5-1 70.8 8.1 71.7 28.1 1 21.3-41.3 30.8-76.5 31.7-55.5 1.5-94.6-11-96-41.3-1.3-27.7 42-47 94.2-48.4 20.7-.5 34.8.3 44.8 1.4 9.9 1 15.5 2 18.5 2zM710 129.7h-97V111h90.4V93H613V76h95.3l-4-17.9h-149v89.4h152.3l2.3-17.9zm-196 22.1 18-4.2V58.2h-54V107L376.6 54l-17.5 4v89.5h54v-47.3zM230 76.2h25.4v53.3H230l-2.5 18h113.7l-2.4-18H313V76.2h25.7l2.4-18H227.5zm-19.4 71.4v-24.9l-45.6 7h-46.6V58.2H60.7v89.4h149.9zM745 185.8a8524 8524 0 0 1-236.5 4.7c-245.5-1.3-451.3-15-464.4-15.9-26.1-2.2-32-4-32-32V59.1c-.3-20.8 1.1-30.8 32-32 13-.9 218.9-14.7 464.4-16 85.1.4 166.5 2.4 237.2 4.8 20.1 54.3 18.2 115.4-.7 169.8z"/></svg>
```

File: entries/l/line6.com.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "Line 6": {
+    "domain": "line6.com",
+    "tfa": [
+      "totp"
+    ],
+    "categories": [
+      "other"
```
Comment (Carlgo11) [inline entries/l/line6.com.json]: Looks like the primary use of an account on line6.com is to buy their products. Is this correct?
```suggestion
      "retail"
```

File: entries/l/line6.com.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "Line 6": {
+    "domain": "line6.com",
+    "tfa": [
+      "totp"
```
Comment (Carlgo11) [inline entries/l/line6.com.json:5]: I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, please attach some screenshots of the 2FA setup process and/or login process.
> **Note** Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment, like this:
screenshot

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by hispanasian** [comment]
> > The image is named `Line6.com.svg` (capital L) instead of `line6.com.svg`. The `"img"` property must be deleted as well here, since it's the default value anyway.
> 
> Thanks for the tips. I took a bit too long to fix the capital `L` here on windows. Wasn't aware I could delete the `img` property, but that explains why it's not on most files. Perhaps it can be removed from the `CONTRIBUTING.MD` example where `img` is explicitly declared unless there's a good reason to keep it there?

Apologies! I completley missed the bullet points under the example. My bad!

File: entries/l/line6.com.json (hunk 2 above)
Comment (hispanasian) [reply entries/l/line6.com.json]: That is correct. Thank you for the guidance on which category to use!

File: entries/l/line6.com.json (hunk 3 above)
Comment (hispanasian) [reply entries/l/line6.com.json:5]: Thanks for the clarification! Unfortunately I have not had luck finding public docs but I can definitely add some screen shots.

Instructions:
1. [Log in](https://line6.com/account)
2. Go to [Two Factor Authentication](line6.com/account/2fa/) on the left menu.
3. Scan the QR code with your chosen 2FA app or enter the provided key.
Screenshot 2023-02-05 234540

Comment (Carlgo11) [reply entries/l/line6.com.json:5]: Thanks!

**Review by Carlgo11 — APPROVED** [review]

**Comment by hispanasian** [comment]
Thanks so much for your patience and guidance with the PR @Carlgo11 and @phallobst  !

**Comment by Carlgo11** [comment]
You're very welcome.
Thanks for the PR!

**Pull request merged**